raiden/tests/unit/test_channelstate.py

```diff
@@ -95,15 +105,15 @@ def create_model(balance, merkletree_width=0):
         amount_locked=0,
         balance=balance,
         distributable=balance,
-        next_nonce=1,
+        next_nonce=len(merkletree_leaves) + 1,
```

Comment:
This is the lowest possible value for the balance proof nonce, since we need at least one `LockedTransfer` message per lock.